### PR TITLE
UI: Fix can't select schedule interval type

### PR DESCRIPTION
**Pull request by utchoang**
### Description

This PR for fix can't select interval type in config backup schedule

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### Screenshots (if appropriate):
image
image

### CODE REVIEW

**Review by rohityadavcloud — APPROVED** [review]
LGTM, does it need to be put on 4.17 branch @utchoang ?

**Comment by utchoang** [comment]
@rohityadavcloud Yes. I think need to put 4.17 branch

**Comment by rohityadavcloud** [comment]
@utchoang thanks go ahead and change the base branch of this PR to 4.17 and rebase your branch (or raise a new PR for 4.17 branch if you've trouble doing the rebasing).

File: ui/src/views/compute/backup/FormSchedule.vue [hunk 1]
```diff
@@ -173,7 +173,8 @@ export default {
       timeZoneMap: [],
       fetching: false,
       actionLoading: false,
-      listDayOfWeek: ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday']
+      listDayOfWeek: ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'],
+      value1: ''
```
Comment (sureshanaparti) [inline ui/src/views/compute/backup/FormSchedule.vue:177]: @utchoang `value1` used anywhere?

Comment (utchoang) [reply ui/src/views/compute/backup/FormSchedule.vue]: @sureshanaparti removed it after rebase to 4.17 branch

**Review by sureshanaparti — APPROVED** [review]

**Review by shwstppr — APPROVED** [review]

**Comment by sureshanaparti** [comment]
Verified schedule interval type selection in 'Configure Backup Schedule' UI form.

**Pull request merged**